Training5/thongnq-training5-task2/src/App.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+import Auth from './pages/Auth';
+
+import Layout from './hoc/Layout';
+import Home from './pages/Home';
+import UserInfo from './pages/UserInfo';
+import UserList from './pages/UserList';
+
+import {BrowserRouter as Router, Switch, Route, Redirect} from "react-router-dom"
+import {useSelector} from "react-redux"
+
+function App() {
+  const user = useSelector(state => state.auth.user);
+
+  return (
+    <div className="App">
+    <Router>
+      <Switch>
+          <Route path="/login" component={Auth} />
+          <Route path={['/app', "/app/users", "/app/users/:userId", "/app/my-info"]}>
```

Comment:
@Thong-Nguyen-Zigvy I think you simply use path ="/app" without exact prop to enough for this case.